website/content/maintainers/templates/governance-elections.md

```diff
@@ -8,6 +8,268 @@ aliases:
+of conduct committee, technical oversight committee, etc. For example, you may
+have a Marketing Committee that handles all work with CNCF marketing, or a
+different body that handles escalated technical disputes.
+
+### Committee Meetings
+
+You need to explicitly describe where, how, and when the Steering Committee will
+meet along with details about where the minutes will be published and how
+community members can contact the Steering Committee or raise issues for
+discussion.
+
+You should consider setting up a cadence of private vs. public meetings with the
+private meetings being for sensitive topics that should not be open to the
+public.
+
+### Committee Members
+
+When a steering committee governance model is first implemented, it usually
+starts with a bootstrap steering committee with members selected by the project
+based on who is currently making project decisions. This group will be
+responsible for running the project, but with a set deadline for how they will
+be replaced by an elected steering committee.
+
+For example, Kubernetes started with a [Bootstrap Governance Committee](https://www.cncf.io/blog/2017/10/16/kubernetes-community-steering-committee-election-results/)
+
+### Decision Process
+
+We recommend using [Lazy Consensus] as the primary decision-making method, but you will
+also need to determine the list of things that requires a vote, such as issuing
+or amending policies, spending, etc. This section will also specify how you
+determine quorum for voting with recommended language found in the template.
+
+You will need to also come up with a process for documenting these decisions
+and have a consistent place where the community can get more information about
+past decisions that were made.
+
+### Getting in touch
+
+Your governance document should specify how community members can raise issues to
+the steering committee for decision. Common ways to do this are via an email
+alias for the Steering Committee, filing an issue in a specific repository, or
+posting in a Steering Committee Slack channel.
+
+### Composition
+
+This section requires some extra work and additional thought put into how you plan
+to handle election terms and replacing Steering Committee members whose terms
+are up for election.
+
+We recommend using staggered 2-year terms where only half of the seats are up
+for election in any given year to maintain continuity. This keeps enough people
+on the Steering Committee to provide context and onboarding for new members.
+
+If you are starting from scratch with a bootstrapped committee, and need a
+transition period, but plan to have staggered 2-year terms, we recommend that for
+the first year of the Steering Committee half of the seats should be elected for a
+1-year term, and the other half should be elected for a 2-year term.
+
+While not recommended, an alternative to staggered terms is that you can 
+have all of the members elected at the same time to serve for a specified term.
```

Comment:
```suggestion
have all of the members elected at the same time to serve for a specified term. 
This can introduce hardship on the newly elected members attempting to pick-up
where the previous Committee left off if there is no formal overlap or handoff.
```